Main.java

```diff
@@ -9,7 +10,40 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+  Scanner input = new Scanner(System.in);
+
+  // ask the user what size pizza they want
+  System.out.println("What is the size of the pizza (in cm):");
+  double size = input.nextDouble();
+
+  // assign variables
+  double diameter = size * 0.50;
+  double labour = 0.75;
+  double power = 0.99;
+
```

Comment:
These numbers should really be constants instead of variables. Remember, constants should always be named using UPPERCASE_LETTERS.